pages/common/mkdir.md

```diff
@@ -9,3 +9,7 @@
 - Create directories recursively (useful for creating nested dirs):
 
 `mkdir -p {{path}}`
+
+- Create multiple directories recursively (note the curly braces, marking directory set):
+
+`mkdir -p dir1/dir2/{dir3/{dir4,dir5},dir6,dir7}`
```

Comment:
Oh, so that's an actual tree with multiple branches? Somehow I feel that's a bit out of scope for tldr...